pom.xml

```diff
@@ -403,7 +404,7 @@
         <version.org.jboss.mod_cluster>1.4.1.Final</version.org.jboss.mod_cluster>
         <version.org.jboss.narayana>5.10.6.Final</version.org.jboss.narayana>
         <version.org.jboss.openjdk-orb>8.1.4.Final</version.org.jboss.openjdk-orb>
-        <version.org.jboss.resteasy>3.14.0.Final</version.org.jboss.resteasy>
+        <version.org.jboss.resteasy>3.15.0.Alpha1</version.org.jboss.resteasy>
```

Comment:
what's the relationship between upgrading MP Config and OpenTracing and bumping RestEasy version?

Reply:
RestEasy has a few Config interface implementations. Those fail to load against with MP Config 2.0 since it introduces some new methods in the Config interface